src/api/layer/linked_accounts.ts

```diff
@@ -27,9 +27,12 @@ export const getLinkedAccounts = get<
   }
 >(({ businessId }) => `/v1/businesses/${businessId}/external-accounts`)
 
-export const confirmConnection = post<
-  Record<string, unknown>,
-  Record<string, unknown>,
+export const confirmAccount = post<
+  never,
+  Partial<{
+    is_unique: boolean
```

Comment:
I don't have an idea but is there a simple way to require one of these us populated? Not high priority.